src/main/resources/io/jenkins/plugins/coverage/CoveragePublisher/config.jelly

```diff
@@ -19,9 +19,11 @@
         <f:entry field="failNoReports" title="Failed if No Reports Found">
             <f:checkbox/>
         </f:entry>
-        <f:entry field="calculateDiffForChangeRequests" title="Calculate diff from change requests">
-            <f:checkbox/>
-        </f:entry>
+        <f:optionalBlock field="calculateDiffForChangeRequests" title="Calculate diff from change requests" inline="true">
+            <f:entry field="failBuildIfCoverageDecreasedInChangeRequest" title="Fail build if code coverage decreased in change request build">
```

Comment:
Maybe we need to move the description to the help file(like [this](https://github.com/jenkinsci/code-coverage-api-plugin/blob/dev/src/main/resources/io/jenkins/plugins/coverage/CoveragePublisher/help-calculateDiffForChangeRequests.html)) and make the title shorter?

How to add a help file for form field: https://wiki.jenkins.io/display/JENKINS/Basic+guide+to+Jelly+usage+in+Jenkins#BasicguidetoJellyusageinJenkins-Helpfiles

Reply:
![image](https://user-images.githubusercontent.com/19176095/65542301-97785b80-df17-11e9-8471-f796c4dafd0e.png)


Thank you